maoni/src/main/java/org/rm3l/maoni/Maoni.java

```diff
@@ -224,12 +226,33 @@ private Maoni(final Builder builder) {
     }
 
     /**
-     * Start the Maoni Activity
+     * Start the Maoni Activity. If screenCapturingFeatureEnabled is true this will take a
+     * screenshot of the activity window
      *
      * @param callerActivity the caller activity
      */
     public void start(@Nullable final Activity callerActivity) {
+        start(callerActivity, callerActivity.getWindow());
+    }
+
+    /**
+     * Start the Maoni Activity. If screenCapturingFeatureEnabled is true this will take a
+     * screenshot of the dialog window
+     *
+     * @param dialog the caller dialog
+     */
+    public void start(@Nullable Dialog dialog) {
+        Activity activity = ContextUtils.scanForActivity(dialog.getContext());
```

Comment:
Added `scanForActivity` in case the `Dialog` context isn't an activity (i.e. `ContextThemeWrapper`)